fuzzers/baby_fuzzer_wasm/src/lib.rs

```diff
@@ -44,7 +44,7 @@ pub fn fuzz() {
     // No concurrency in WASM so these accesses are not racing.
     unsafe {
         RegistryBuilder::register::<MapFeedbackMetadata<u8>>();
-        RegistryBuilder::register::<ExecutionCountRestartHelperMetadata>();
+        RegistryBuilder::register::<StdRestartHelper >();
```

Comment:
Wait, that's a different RestartHelper alltogether? That's the one that counts the target executions(?)

Reply:
Can WASM even restart? :D

Reply:
> Wait, that's a different RestartHelper alltogether? That's the one that counts the target executions(?)

ExecutionCountRestartHelperMetadata was not even used in mutationalstage (the only stage that uses this metadata)

previously it was `Ok(true)` = always restart no matter what.
I then changed it to restart 3 times at max

Reply:
We used ExecutionCountRestartHelper to resume the mutational stage (I still think this is more correct btw) but it performed worse in fuzzbench